providers/base.ts

```diff
@@ -216,9 +216,14 @@ export abstract class ReleaseLookup<Provider extends MetadataProvider, RawReleas
 			}
 
 			const releaseType = this.provider.entityTypeMap['release'];
-			if (entity.type !== releaseType) {
+			if (
+				!(
+					Array.isArray(releaseType) ? releaseType.includes(entity.type) : entity.type === releaseType
+				)
+			) {
```

Comment:
```suggestion
			if (
				Array.isArray(releaseType) ? !releaseType.includes(entity.type) : entity.type !== releaseType
			) {
```
Proabably just a matter of preference, but this feels more readable to me.
What do you think? Feel free to disregard.

Reply:
I don't mind either way, so as you prefer. :)